src/OmniSharp/Models/v1/HighlightRequest.cs

```diff
@@ -0,0 +1,89 @@
+namespace OmniSharp.Models
+{
+    public class HighlightRequest : Request
+    {
+        /// <summary>
+        ///   Specifies which lines to highlight.
+        ///   If none are given, highlight the entire
+        ///   file.
+        /// </summary>
+        public int[] Lines { get; set; }
+        /// <summary>
+        ///   Specifies which projects to highlight for.
+        //    If none are given, highlight for all the projects.
+        /// </summary>
+        public string[] ProjectNames { get; set; }
+        /// <suimmary>
+        ///   Request comment classifications
+        /// </summary>
+        /// <remarks>
+        /// Defaults to true
+        /// </remarks>
+        public bool WantComments { get; set; } = true;
```

Comment:
What would that array be a type of? just string?  I thought about an array at first, but we only get strings back from classifier, hence I went with this request approach.